### PR TITLE
Create database seeding script

### LINKED ISSUE #134: Add DB Seeding Functionality
### Description:
Implement functionality to seed the database with initial data. This will populate essential tables with predefined data to ensure the application's initial state is consistent and usable.

### Tasks:
1. Create seed data scripts for essential database tables (e.g., users, mentees, profiles).
2. Implement a mechanism to execute these scripts automatically during application startup or through a dedicated seeding command.
3. Ensure that seeded data is consistent and reflects typical usage scenarios.

### Acceptance Criteria:
- Database tables are populated with initial data upon application startup or upon running a seeding command.
- Seeded data includes necessary entries for basic functionality and testing.
- Seeding operation logs successes and failures appropriately.
- README.md includes clear instructions on how to seed the database.

### PR DESCRIPTION
**Pull request by dsmabulage**
## Purpose

The purpose of this PR is to fix #134

## Goals

1. Add db seeding for initial db with sample values

## Approach

1. Used fakers to generate random values.
2. All the tables were populated with the relations.
3. [localhost:3000/api/seeds/seed](http://localhost:3000/api/seeds/seed) will seed the db.

### Screenshots

image

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: src/routes/seeds/seeds.route.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import express from 'express'
+import { seedDbController } from '../../controllers/seed.controller'
+
+const seedRouter = express.Router()
+
+seedRouter.get('/seed', seedDbController)
```
Comment (anjula-sack) [inline src/routes/seeds/seeds.route.ts:6]: We don't need an endpoint here. Just make this  a script where we can execute using something like `npm run seed`

**Review by anjula-sack — CHANGES_REQUESTED** [review]

**Review by anjula-sack — CHANGES_REQUESTED** [review]
Can you check the lint issues? @dsmabulage

File: src/routes/seeds/seeds.route.ts (hunk 1 above)
Comment (dsmabulage) [reply src/routes/seeds/seeds.route.ts]: resolved

**Comment by dsmabulage** [comment]
> Can you check the lint issues? @dsmabulage

fixed that

**Review by anjula-sack — APPROVED** [review]
Good job! @dsmabulage

**Pull request merged**